### DISCUSSION TITLE
Any idea how to add sbat section required for all new efi binaries

### DISCUSSION THREAD

**Discussion by MuthuvelKuppusamy** [tianocore/edk2#2907] [General]
Hi,
I would like to know more about, how to add the sbat section in all the edk2 build efi file automatically.

Thanks,
Muthuvel K

**Comment by spbrogan** [comment]
I assume you are talking about a feature of the "shim" https://github.com/rhboot/shim/blob/main/SBAT.example.md#:~:text=If%20an%20SBAT%20variable%20is%20set%2C%20binaries%20validated,the%20validation%20result%20is%20logged%20only%2C%20not%20enforced

you would probably do better following up in that project.
